packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -114,6 +116,30 @@ function isClientMessage(message: ISequencedDocumentMessage | IDocumentMessage):
 	}
 }
 
+/**
+ * Like assert, but logs only if the condition is false, rather than throwing
+ * @param condition - The condition to attest too
```

Comment:
Nitpick: (a bit out of scope for this pr) Consider mentioning this is supposed to be some kind of truthy/falsey variable. Also looks like its not invoked like a function but it feels like you can provide a function as the "condition" which is kinda confusing.

Reply:
Yeah, good callouts. I'll take a look at how this is used in this file and narrow the behavior and docs to match.